src/Tizen.Multimedia.Camera/Camera/Camera.cs

```diff
@@ -57,13 +59,53 @@ public Camera(CameraDevice device)
         {
             Native.Create(device, out _handle).ThrowIfFailed("Failed to create camera instance");
 
+            Initialize();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Camera"/> class with <see cref="CameraDeviceType"/>.
+        /// </summary>
+        /// <param name="device">The camera device to access.</param>
+        /// <param name="type">Indicates whether this is network camera or not.</param>
+        /// <exception cref="ArgumentException">Invalid CameraDevice or CameraDeviceType.</exception>
+        /// <exception cref="NotSupportedException">The camera feature is not supported.</exception>
```

Comment:
I think InvalidOperationException should be added.

Reply:
Ok I'll fix it.